demo/index.html

```diff
@@ -34,11 +35,16 @@
     margin: 0;
     padding: 0;
   }
+  url-bar {
+    background-color: white;
```

Comment:
This seems like something that should be a default in the element, and can be overridden with a custom property.